app/models/cid.rb

```diff
@@ -0,0 +1,127 @@
+class Cid
+  include Neo4j::ActiveNode
+
+  property :documentType
+  property :contentID
+  property :name
+  property :description
+  property :pagerank
+  property :title
+
+  has_many :out, :eligibilities, type: :HAS_ELIGIBILITY, model_class: :Eligibility
+
+  def self.where_not_ineligible(attributes)
+    Eligibility.
+      all.
+      reject { |eligibility| eligibility.ineligible?(attributes) }.
+      map { |eligibilty| eligibilty.cids }.
+      flatten.
+      compact
+  end
+
+  def self.test_toy_example
```

Comment:
I know this is only a spike, but is this perhaps better extracted and expressed as a rake task or a spec just for ease of running it?

Reply:
True, I've moved it all to one giant test, obvs this'd want splitting up at some point but not yet